apps/cowswap-frontend/src/common/analytics/utils.ts

```diff
@@ -0,0 +1,19 @@
+import { CowAnalytics } from '@cowprotocol/analytics'
+import { debounce } from '@cowprotocol/common-utils'
+
+import { Field } from 'legacy/state/types'
+
+import { CowSwapCategory } from './types'
+
+/**
+ * Creates a debounced function that sends analytics events for trade amount changes
+ */
+export function createDebouncedTradeAmountAnalytics(cowAnalytics: CowAnalytics) {
```

Comment:
The function probably should be a part of `modules/trade` since it's logically connect only to trade widgets.

Reply:
Addressed